nixos/modules/services/hardware/undervolt.nix

```diff
@@ -8,6 +8,18 @@ let
     if (limit == null && window == null) then null
     else assert asserts.assertMsg (limit != null && window != null) "Both power limit and window must be set";
       "${toString limit} ${toString window}";
+  hasCliArgs = (
+    (cfg.coreOffset != null) ||
+    (cfg.gpuOffset != null) ||
+    (cfg.uncoreOffset != null) ||
+    (cfg.analogioOffset != null) ||
+    (cfg.tempAc != null) ||
+    (cfg.tempBat != null) ||
+    (cfg.p1.limit != null) ||
+    (cfg.p1.window != null) ||
+    (cfg.p2.limit != null) ||
+    (cfg.p2.window != null)
+  );
```

Comment:
Why some flags such as `temp` and `turbo` are not listed here?

Reply:
Because one of the args that are listed here need to be present for the undervolt utility to actually work. `temp` or `turbo` are insufficient.

Reply:
Actually, it turns out that just setting turbo is sufficient for the utility to work. Additionally, there were some issues with the way that temperature was handled, which are now fixed.